controllers/release/adapter.go

```diff
@@ -804,7 +967,7 @@ func (a *adapter) validatePipelineDefined() *controller.ValidationResult {
 			return &controller.ValidationResult{Err: err}
 		}
 		if releasePlanAdmission.Spec.Pipeline == nil {
-			errString := "releasePlan and releasePlanAdmission both have no pipeline. Each Release should define a tenant pipeline, managed pipeline, or both"
+			errString := "releasePlan has no pipeline or target. Each Release should define a tenant pipeline, managed pipeline or both"
```

Comment:
This string should have remained as is

Reply:
Fixed

Reply:
I believe you switched the errStrings here

Reply:
Fixed